diff -urN libsndfile-1.0.25/doc/libsndfile.css libsndfile-1.0.25-patched/doc/libsndfile.css
diff -urN libsndfile-1.0.25/man/sndfile-deinterleave.1 libsndfile-1.0.25-patched/man/sndfile-deinterleave.1
diff -urN libsndfile-1.0.25/man/sndfile-metadata-set.1 libsndfile-1.0.25-patched/man/sndfile-metadata-set.1
diff -urN libsndfile-1.0.25/programs/sndfile-play.c libsndfile-1.0.25-patched/programs/sndfile-play.c
--- libsndfile-1.0.25/programs/sndfile-play.c	2011-03-28 07:15:31.000000000 +0200
+++ libsndfile-1.0.25-patched/programs/sndfile-play.c	2013-08-26 21:44:37.000000000 +0200
@@ -58,7 +58,7 @@
 	#include 	<sys/soundcard.h>
 
 #elif (defined (__MACH__) && defined (__APPLE__))
-	#include <Carbon.h>
+//	#include <Carbon.h>
 	#include <CoreAudio/AudioHardware.h>
 
 #elif defined (HAVE_SNDIO_H)
diff -urN libsndfile-1.0.25/src/version-metadata.rc libsndfile-1.0.25-patched/src/version-metadata.rc
